test/EnhancedScatterChartMock.ts

```diff
@@ -7,7 +7,7 @@ export interface SelectionState {
 
 export class EnhancedScatterChartMock extends EnhancedScatterChart{
 
-    protected override telemetryTrace(): void {
-        this.externalImageTelemetryTraced();
+    protected telemetryTrace(): void {
+        //this.externalImageTelemetryTraced();
```

Comment:
why commented?

Reply:
before refactoring of telemetry, the mock included dummy telemetry trace method test with, now telemetry is separate service and new test includes only url link checks

Reply:
remove empty function